main/acle.md

```diff
@@ -9712,6 +9728,42 @@ Lookup table read with 4-bit indices.
   svint16_t svluti4_lane[_s16_x2](svint16x2_t table, svuint8_t indices, uint64_t imm_idx);
 ```
 
+### SVE2 Multi-vector AES and 128-bit polynomial multiply long instructions
+
+The specification for SVE2 Multi-vector AES and 128-bit polynomial multiply long instructions is in
+[**Alpha** state](#current-status-and-anticipated-changes) and might change or be
+extended in the future.
+
+#### AESE, AESD, AESEMC, AESDIMC
+
+Multi-vector Advanced Encryption Standard instructions
+
+```c
+  // Only if __ARM_FEATURE_SVE_AES2 != 0
+
+  svuint8x2_t    svaese_lane[_u8_x2]     (svuint8x2_t op1, svuint64_t op2, uint64_t index);
+  svuint8x4_t    svaese_lane[_u8_x4]     (svuint8x4_t op1, svuint64_t op2, uint64_t index);
+  svuint8x2_t    svaesd_lane[_u8_x2]     (svuint8x2_t op1, svuint64_t op2, uint64_t index);
+  svuint8x4_t    svaesd_lane[_u8_x4]     (svuint8x4_t op1, svuint64_t op2, uint64_t index);
+  svuint8x2_t    svaesemc_lane[_u8_x2]   (svuint8x2_t op1, svuint64_t op2, uint64_t index);
+  svuint8x4_t    svaesemc_lane[_u8_x4]   (svuint8x4_t op1, svuint64_t op2, uint64_t index);
+  svuint8x2_t    svaesdimc_lane[_u8_x2]  (svuint8x2_t op1, svuint64_t op2, uint64_t index);
+  svuint8x4_t    svaesdimc_lane[_u8_x4]  (svuint8x4_t op1, svuint64_t op2, uint64_t index);
+```
+
+#### PMULL, PMLAL
+
+Multi-vector 128-bit polynomial multiply long instructions
+
+``` c
+  // Only if __ARM_FEATURE_SVE_AES2 != 0
+
+  // Variants are also available for:
+  // _s64x2, _f64x2
+  svuint64x2_t svpmull[_u64x2](svuint64_t zn, svuint64_t zm);
```

Comment:
I believe that we also should have the _n version, like I see in the llvm implementation.
I am proposing something like this:

svuint64x2_t svpmull_u64x2(svuint64_t zn, svuint64_t zm);
svuint64x2_t svpmull[_n]_u64x2(svuint64_t zn, uint64_t zm);

svuint64x2_t svpmlal[_u64x2](svuint64x2_t zda, svuint64_t zn, svuint64_t zm);
svuint64x2_t svpmlal[_n_u64x2](svuint64x2_t zda, svuint64_t zn, uint64_t zm)